pkg/registry/endpoint/etcd/etcd.go

```diff
@@ -33,7 +33,7 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against endpoints.
-func NewStorage(h tools.EtcdHelper) *REST {
+func NewStorage(s tools.StorageInterface) *REST {
```

Comment:
You're asking the guy who chose "lmctfy" for naming advice?

On Mon, Aug 3, 2015 at 12:53 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/registry/endpoint/etcd/etcd.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11806#discussion_r36062802>
> :
>
> > @@ -33,7 +33,7 @@ type REST struct {
> >  }
> >
> >  // NewStorage returns a RESTStorage object that will work against endpoints.
> > -func NewStorage(h tools.EtcdHelper) *REST {
> > +func NewStorage(s tools.StorageInterface) *REST {
>
> BTW - @thockin <https://github.com/thockin> do you have any proposal what
> names those should have (I'm very bad in "inventing" names)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11806/files#r36062802>
> .
>